app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,20 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %form.sign_in(action=login_path method="post")
+        = hidden_field_tag :authenticity_token, form_authenticity_token
```

Comment:
Instead of this, you should use `= form_tag login_path do` because Rails form helpers yield advantage like that and produce authenticity tokens under the hood.